src/components/common/NetworkSelector/index.tsx

```diff
@@ -27,6 +27,30 @@ const MenuWithTooltip = forwardRef<HTMLUListElement>(function MenuWithTooltip(pr
   )
 })
 
+type NetworkLogos = Record<string, string>
+
+export const NetworkLogos: NetworkLogos = {
+  eth: '/images/networks/mainnet.svg',
+  bnb: '/images/networks/bnb.svg',
+  oeth: '/images/networks/optimism.svg',
+  gno: '/images/networks/gno.png',
+  matic: '/images/networks/polygon.svg',
+  aurora: '/images/networks/aurora.svg',
+  base: '/images/networks/base.svg',
+  ['base-gor']: '/images/networks/base.svg',
+  zkevm: '/images/networks/polygon.svg',
+  zksync: '/images/networks/zksync.svg',
+  celo: '/images/networks/celo.svg',
+  gor: '/images/networks/gor.svg',
+  arb1: '/images/networks/arb.svg',
+  avax: '/images/networks/avax.svg',
+  sep: '/images/networks/sep.png',
```

Comment:
Could you use the `chains.ts` module for the keys?
Also it would make sense to keep this config in a global module because it's going to be used in all network indictors.

Reply:
I extracted them to the `@/config/chains` file since thats where we access the `chains.ts` module as well.